module/VuFind/src/VuFindTest/Integration/MinkTestCase.php

```diff
@@ -126,15 +125,16 @@ protected function changeYamlConfigs($configs, $replace = [])
      */
     protected function changeConfigFile($configName, $settings, $replace = false)
     {
+        $resolver = new \VuFind\Config\PathResolver();
```

Comment:
I wonder if it would be better to have a single instance of this accessed via a getter rather than building it on the fly every time -- especially if we eventually need a factory to build this with proper defaults.

Reply:
Added.